Article/Article.js

```diff
@@ -3,27 +3,25 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');      //this.expandButton = domElement.querySelectorAll('.expandButton');  
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle());
+
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    this.domElement.classList.toggle('article-open');
   }
 }
 
 /* START HERE: 
-
 - Select all classes named ".article" and assign that value to the articles variable.  
-
 - With your selection in place, now chain .forEach() on to the articles variable to iterate over the articles NodeList and create a new instance of Article by passing in each article as a parameter to the constructor.
-
 */
 
-let articles;
\ No newline at end of file
+let articles = document.querySelectorAll('article').forEach(article => new Article(article));
```

Comment:
Remember, you are trying to grab all the elements with the class name article. And how do you tell the query selector that you are looking for class names?